media-video/baka-mplayer/baka-mplayer-2.0.4.ebuild

```diff
@@ -31,7 +31,7 @@ DEPEND="${RDEPEND}"
 
 S="${WORKDIR}/Baka-MPlayer-${PV}"
 
-PATCHES=( "${FILESDIR}/${P}-gcc5.patch" )
+PATCHES=( "${FILESDIR}/${P}-gcc5.patch" "${FILESDIR}/${P}-mpv23.patch" )
```

Comment:
PS: If this is a runtime fix, then you will have to make a revbump to this package, drop stable keywords. no changes to the existing ebuild.

Reply:
The patch was just for a runtime bug so I did the revbump without stable keywords and reverted the 2.0.4 ebuild.